benchmarks/portable/chaos.py

```
@@ -1118,64 +1541,121 @@ def scenario_priority_starvation(system: str, job_count: int = 20) -> dict:
+                "DATABASE_URL": pg_url("awa_bench"),
+                "SCENARIO": "worker_only",
+                "WORKER_COUNT": "3",
+                "JOB_DURATION_MS": "2000",
+                "RESCUE_INTERVAL_SECS": "5",
+                "HEARTBEAT_STALENESS_SECS": "15",
+            },
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
     elif system == "river":
         worker = subprocess.Popen(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)